snap/hooks/configure

```diff
@@ -29,9 +23,9 @@ if [[ -z "$http_address" ]]; then
 fi
 
 # gRPC address to send profiles and symbols to.
-store="$(snapctl get store-address)"
+remote_store="$(snapctl get remote-store-address)"
 if [[ -z "$store" ]]; then
```

Comment:
```suggestion
if [[ -z "$remote_store" ]]; then
```

Reply:
@kakkoyun ok this was a more sensible place to put the comment ;-)

Reply:
Great catch!